py4syn/epics/ImageHDFClass.py

```diff
@@ -0,0 +1,111 @@
+"""Image HDF Class
+
+Python Class for Save HDF images .
+
+:platform: Unix
+:synopsis: Python Class for HDF images.
+
+.. moduleauthor:: Gabriel Fedel <gabriel.fedel@lnls.br>
+.. note:: 06/02/2017 [gabrielfedel]  first version released
+"""
+import os
+
+import numpy as np
+import h5py
+from py4syn.epics.StandardDevice import StandardDevice
+from py4syn.epics.ICountable import ICountable
+
+class ImageHDF(StandardDevice, ICountable):
```

Comment:
Hello Hugo. My first try was like your suggestion, but i not found an way to start the 2 constructors (StandardDevice and ImageHDF) on subclasses... Do you know if there is an way to do this?  Or, maybe, another solution (remove from ImageHDF constructor and create a simple method)...